src/rgb/contract/mod.rs

```diff
@@ -18,9 +18,13 @@ pub mod test_helpers {
     use crate::paradigms::client_side_validation::{CommitEncode, Conceal};
```

Comment:
I tried putting the test helper module at the ned of mod.rs file. But this then gives this error at the macro calls
```
cannot find macro `test_encode` in this scope
have you added the `#[macro_use]` on the module/import?
```
Not sure how exactly to solve this. Putting the module at the top of `mod.rs` file removes this error.

Reply:
Right. Can you move the whole mode into separate file test helpers and just `use` it from the same place in `mod.rs`? This will work (your macro declaration has to be imported before those stuff that uses the macro)

Reply:
Will do that..